task 5.py

```diff
@@ -59,11 +83,26 @@ def check_type(t):
         phone = input('What is your phone number?\n')
         anounce = AddAnnouncement(text, phone)
         anounce.add_record()
+    elif t == 4:
+        file = input('Enter file name\n')
```

Comment:
Also, use in the next iterations the error handling. Like try: except:
For example, you can catch errors like a file does not exist or invalid format for date in ADs or an invalid phone number and ask the user to try again to input the data.